tests/conftest.py

```diff
@@ -135,7 +136,7 @@ def form_element_help_text(db, survey_form_entry):
 @pytest.mark.django_db
 def survey(db, location, study):
     survey = Survey.objects.create(
-        time_stop=datetime.now(tz=None),
+        time_stop=datetime.now(tz=get_current_timezone()),
```

Comment:
Another good call.

Reply:
Pytest pointed it out as a warning! Thank you pytest. 🙏🏻